harvardcards/templates/_collection_view.html

```diff
@@ -27,10 +27,18 @@ <h2 class="courseTitle">{{ collection.title }}</h2>
         {% for deck in collection.decks %}
             <li class="deck">
                 <dl>
+                    {% if deck.id < 0 %}
+                    <dt><a href="{% url 'allCards' deck.id|stringformat:'+d'|slice:'1:'  %}">{{ deck.title }}</a></dt>
```

Comment:
Since ```deck.id``` in this context is the negative collection_id, this ```deck.id|stringformat:'+d'|slice:'1:'``` can be simplified to ```collection.id``` since we already have the collection ID.